docs/js/search.js

```diff
@@ -243,6 +243,21 @@
         title: 'JavaScript > Waves',
         tags: 'wave javascript js',
         body: 'Waves is an external library that weve included in Materialize to allow us to create the ink effect outlined in Material Design'
+      }, {
+        href: 'css-transitions.html',
```

Comment:
It seems better to arrange values of `href` in ABC order.

Reply:
that was what I previously thought... which do you think is better? sort by "title" or by "href"? or sort according to the sidenav in the documentation?

Reply:
@Smankusors 
It's difficult... I think it's better to sort by `href` because we can check the equality by comparing here with files in `page-contents` directory.

Reply:
hmm what do you think? @DanielRuf

Reply:
Good question. If we use the href we should ensure that the titles are the same or start with the same letters.

Personally I would find it better to use the titles and show the titles when I search something.

Reply:
@Smankusors Please mention to me after making a new commit. I want to approve this soon! ❤️

Reply:
@nekonenene I will, but I think next month, I got a thesis to work on 😅

Reply:
@Smankusors I got it. No problem, thanks!

Reply:
now.... I'm actually confused because @nekonenene wants this sort by `href`, while @DanielRuf  wants this sort by `title`.

For me.... well... actually doesn't matter because I can use Ctrl + F 😆 

Or... sort by title could make this better.... Are you okay with this btw? @nekonenene

Reply:
In search results I would expect to see a logical visual order and what I see is mainly the title. Frameworks and shops normally order alphabetically or by score / match percentage (similarity). href (not a visual identifier) would not be the best UX in my opinion, for example when the titles differ and you have to search for the correct letter because it is differently sorted / ordered.

Reply:
@Smankusors Sorry for the late reply! I support Daniel's opinion.